### ISSUE TITLE
[1.3.3-beta.2] Email.send not working unless headers object provided.

### ISSUE THREAD

**Issue by saimeunt** [meteor/meteor#7123]
Looks like this new feature : **Insert a Date header into emails by default:** https://github.com/meteor/meteor/pull/6916/files leads to a serious regression if `options.headers` is `undefined`.

https://github.com/meteor/meteor/blob/release-1.3.3/packages/email/email.js#L181

**Comment by saimeunt** [comment]
BTW, quick fix would be:

```js
const emailSendCopy = Email.send;
Email.send = (options) => {
  Object.assign(options, { headers: {} });
  emailSendCopy(options);
};
```

**Comment by abernix** [comment]
Thanks for reporting this, @saimeunt. I would expect it to be fixed in the next beta.

If your mail sending is all done from a single entry point, just passing an empty headers object is also an easy fix:
```js
Email.send({
  from: "foo@example.com",
  to: "bar@example.com",
  // more usual properties here
  headers: {},
});
```

**Comment by abernix** [comment]
This has already been fixed with 6e071838af0c9594cdbf9b23ded7ef5806510a86 and should be in the next beta.  Also PR #7128 has been submitted which would have caught this mistake!

**Comment by abernix** [comment]
Thanks for reporting this.  This should be fixed in the new beta that just came out.  You can update to it using `meteor update --release 1.3.3-beta.2`.

Thanks for reporting this and please write back if you're still having any problems!
